parquet-hadoop/src/main/java/org/apache/parquet/crypto/AesCipher.java

```diff
@@ -90,6 +102,10 @@
 
   // Update last two bytes with new page ordinal (instead of creating new page AAD from scratch)
   public static void quickUpdatePageAAD(byte[] pageAAD, short newPageOrdinal) {
+    if (newPageOrdinal < 0) {
```

Comment:
Since this is a public method, can we validate pageAAD also?

Reply:
Ok, I'll add this.

Reply:
I agree on validating the arguments is important. But the proper exception to be thrown for a `null` is a `NullPointerException` that would be thrown at line 134 anyway. If you really want to validate the argument for `null` at the first line of the method I would suggest using `java.util.Objects.requireNonNull(Object)`.